cmd/kwil-admin/cmds/whitelist/cmd.go

```diff
@@ -1,14 +1,13 @@
-package peers
+package whitelist
 
 import "github.com/spf13/cobra"
 
 var peersCmd = &cobra.Command{
-	Use:     "peers",
-	Short:   "manages the node's peers",
-	Aliases: []string{"peer"},
+	Use:   "whitelist",
+	Short: "manages the node's whitelist peers",
```

Comment:
Small nit, but can you make sure you have consistent grammar for all of your user-facing docs? For example, in this command, you don't capitalize the first letter of the sentence, nor do you have punctuation. In another command, you capitalize the first letter, but don't have punctuation, and in another command, you do have punctuation and you capitalize the first letter.

We should make sure to have proper grammar is user facing docs like this, so please capitalize the first letter, add punctuation, etc. Also, make sure that they are complete sentences: e.g. "The `peer` command is used to manage a node's peer whitelist".